api/all/src/main/java/io/opentelemetry/api/DefaultOpenTelemetry.java

```diff
@@ -24,6 +24,11 @@
 @ThreadSafe
 public class DefaultOpenTelemetry implements OpenTelemetry {
   private static final Object mutex = new Object();
+  @Nullable private static volatile OpenTelemetry globalOpenTelemetry;
```

Comment:
static looks good, I would keep the instance fields where they were and move the constructor to be just below them, instead of moving them above the statics

Reply:
I personally prefer this order, with the fields all above all the methods. Does the google style guide specify one way or another? If not, we should have a documented order, so we don't need to think about it. :)

Reply:
I always have (in my code) fields first then methods.

Reply:
@anuraaga you ok with switching to this style? If so, I can add it to the contributing doc, so it's not a point of debate. I checked the google style guide and it explicitly doesn't specify this. Only that things are in a "logical" order.

Reply:
Oops sorry for missing replying

I've always found statics before non-statics to be easier to reason about - lifecycle and end-user usage is totally different. The instance fields can't be used in the static methods no matter what so they seem distracting to be grouped with them. I wouldn't block the change since 2/3 wins but please give it one more consideration given this reasoning :)